app/controllers/bibliographic_controller.rb

```diff
@@ -92,15 +92,19 @@ def bib_holdings
   end
 
   def bib_items
-    records = adapter.get_items_for_bib(bib_id_param)
-    if records.nil? || records.empty?
-      render plain: "Record #{params[:bib_id]} not found or suppressed", status: 404
-    else
-      respond_to do |wants|
-        wants.json  { render json: MultiJson.dump(add_locator_call_no(records)) }
-        wants.xml { render xml: '<todo but="You probably want JSON anyway" />' }
-      end
+    item_keys = ["id", "pid", "perm_location", "temp_location"]
+    records = adapter.get_items_for_bib(bib_id_param).holding_summary(item_key_filter: item_keys)
```

Comment:
nit-picking: Maybe rename to `record` (singular) since we only handle one record ?

Reply:
what about `holding_summary`?